Interview_Prep_App/native_kotlin/clean-architecture/src/main/assets/answers.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
why did you put the .jsons here? Are you still getting them from the github repo later?